app/controllers/user_controller.rb

```diff
@@ -20,9 +25,15 @@ def signup
   def update_user
     current_user = User.find(params[:id])
     payload = JSON.parse(request.body.read)
+    error_messages = Validation.update_user_validate(payload)
+    if !error_messages.empty?
```

Comment:
You can switch to `if error_messages.present?` to remove the negation. Make the code slightly more readable.